enterprise/frontend/src/metabase-enterprise/transforms/pages/TransformPage/TargetSection/TargetSection.tsx

```diff
@@ -82,7 +84,14 @@ function TargetInfo({ transform }: TargetInfoProps) {
           <TargetItemLink
             label={target.schema}
             icon="folder"
-            to={getBrowseSchemaUrl(database.id, target.schema)}
+            to={
+              table ? getBrowseSchemaUrl(database.id, table.schema) : undefined
+            }
+            tooltip={
+              table?.schema
+                ? undefined
+                : t`This schema will be created when the transform runs`
```

Comment:
I think this logic is incorrect. The schema can be already there when the `table` is not created yet. I think there is no way but to fetch the list of schemas (with `include_hidden: true`) and compare

Reply:
Please add tests for this (can be a unit test)